airflow/gcp/operators/dataflow.py

```diff
@@ -393,7 +393,7 @@ def __init__(
             py_options: Optional[List[str]] = None,
             dataflow_default_options: Optional[dict] = None,
             options: Optional[dict] = None,
-            py_interpreter: str = "python2",
+            py_interpreter: str = "python3",
```

Comment:
I don't disagree with this change, but this is stretching the definition of "in this code base". The entirety of Airflow could be Py3 only and this could stay as python2 and it would work.

Reply:
This should be done in a separate PR (to create an entry in the changelog) and we should add a note in the UPDATING.MD file.
https://github.com/apache/airflow/pull/5602
Here is the discussion where changing this parameter to Python3 was discussed, although now the situation may be different.
@jaketf  @lgajowy WDYT?

Reply:
I will create a separate PR to adjust the change in Dataflow, and update the changelog

Reply:
See the separate PR #6945 https://github.com/apache/airflow/pull/6945